### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by bpappin** [halilozercan/compose-richtext#83]
Cause 1: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find org.jetbrains.compose.runtime:runtime:1.2.0-alpha01-dev683.
Searched in the following locations:
  - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/runtime/runtime/1.2.0-alpha01-dev683/runtime-1.2.0-alpha01-dev683.pom
  - https://repo.maven.apache.org/maven2/org/jetbrains/compose/runtime/runtime/1.2.0-alpha01-dev683/runtime-1.2.0-alpha01-dev683.pom
Required by:
    project :app > project :ui > com.halilibo.compose-richtext:richtext-commonmark:0.12.0 > com.halilibo.compose-richtext:richtext-commonmark-android-debug:0.12.0
    project :app > project :ui > com.halilibo.compose-richtext:richtext-ui-material:0.12.0 > com.halilibo.compose-richtext:richtext-ui-material-android-debug:0.12.0
    project :app > project :ui > com.halilibo.compose-richtext:richtext-commonmark:0.12.0 > com.halilibo.compose-richtext:richtext-commonmark-android-debug:0.12.0 > com.halilibo.compose-richtext:richtext-ui:0.12.0 > com.halilibo.compose-richtext:richtext-ui-android-debug:0.12.0

**Comment by halilozercan** [comment]
There is a problem with using _unstable_ releases of Jetbrains Compose. If you still want to use the 0.12.0 release, please add the following to the same project that you are using RichText 

```
repositories {
  maven { url = uri("https://maven.pkg.jetbrains.space/public/p/compose/dev") }
}
```

**Comment by halilozercan** [comment]
This should be fixed by #85

**Comment by SilverEnd** [comment]
This issue happens on `0.14.x` too and i believe its because of this change https://github.com/halilozercan/compose-richtext/pull/91/files#diff-4eb43b8eb96c3ff852e0f833b73a226c6505296ed2478be35a9f58114f895a65R3
